src/python/bot/fuzzers/libfuzzer.py

```diff
@@ -634,12 +634,30 @@ def merge(self,
     # TODO(flowerhack): Integrate some notion of a merge timeout.
     self._push_corpora_from_host_to_target(corpus_directories)
 
+    additional_args = copy.copy(additional_args)
+    if additional_args is None:
+      additional_args = []
+
+    target_tmp_dir = None
+    if tmp_dir:
+      # Tmp dir may have artifacts, e.g. merge control file for two step merge.
+      target_tmp_dir = self._corpus_target_subdir(os.path.basename(tmp_dir))
+      self.fuzzer.device.rm(target_tmp_dir, recursive=True)
+      self.fuzzer.device.store(tmp_dir, target_tmp_dir)
+      for i, argument in enumerate(additional_args):
+        additional_args[i] = argument.replace(tmp_dir, target_tmp_dir)
```

Comment:
is this to replace the merge_control_file arg? since this is in a libfuzzer specific code, we can just change the interface to add an argument for it to make it easier to deal with in all the subclasses which need to map it into a different path. 

i.e.

```python
def merge(self, ..., merge_control_file=None):
  if merge_control_file:
     # in subclasses map path as necessary.
     additional_arguments.append(constants.MERGE_CONTROL_FILE_ARGUMENT + merge_control_file)
    ...
```